nautobot_netbox_importer/tests/fixtures/nautobot-v2.2/3.7.custom/summary.json

```diff
@@ -9520,12 +9520,16 @@
                         "model_flags": "DiffSyncModelFlags.NONE",
                         "diffsync": "NetBoxAdapter \"NetBox\"",
                         "id": "03d4a2c5-e08b-5cc1-924d-c401194d393c",
-                        "manufacturer_id": "9568e4c1-fa0c-5c43-9606-e25166e67765",
-                        "model": "Half U Device Type",
-                        "created": "2024-02-15 09:18:48.416000+00:00",
+                        "manufacturer_id": "None",
+                        "model": "None",
+                        "created": "None",
```

Comment:
My only guess is the tests aren't deterministic. I don't think I changed anything that would impact this.